.jshintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "bitwise": false,
+  "freeze": true,
+  "globals": false,
+  "maxdepth": 3,
+  "maxparams": 4,
```

Comment:
Causes "modules/servo-scanner.js: line 3, col 22, This function has too many parameters. (5) (W072)"